tagger.py

```diff
@@ -26,43 +30,62 @@
 """
 HORIZONTAL_RULE = "-" * 140
 
-parser = argparse.ArgumentParser(description="Organize music files")
-parser.add_argument(
+
+@click.command()
+@click.option(
     "--source_path",
-    dest="source_path",
-    type=str,
-    help="The parent folder of the directory tree containing all music files you want to organize.",
+    help="The parent folder of the directory tree containing all music files you want to organize",
+    type=click.Path(exists=True, file_okay=False),
     required=True,
 )
-parser.add_argument(
+@click.option(
     "--destination_path",
-    dest="destination_path",
-    type=str,
-    help="The parent folder where the music files are going to be organized.",
+    help="The parent folder where the music files are going to be organized",
+    type=click.Path(exists=True, file_okay=False),
```

Comment:
The destination folder doesn't need to exist, cause it will be created recursively while translating the organizing_template/organizing_tags into directory names and it would be really painfull to create directory by directory.